docs/release-notes/2.40.0.md

```diff
@@ -0,0 +1,102 @@
+
+# Percona Monitoring and Management 2.40.0
+
+
+| **Release date:** | [Month DD, YYYY]                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+<!---
+
+!!! caution alert alert-warning "Important/Caution"
+    Crucial points that need emphasis:
+
+    - Important: A significant point that deserves emphasis.
+    - Caution: Used to mean 'Continue with care'.
+
+--->
+
+## Release Highlights
+
+### Inventory improvements
+
+#### Ability to update Labels for already added Services to PMM  
+PMM-11423,PMM-11148 : In the past, the only solution to fixing incorrect labels was to remove and re-add the Service to PMM. However, now it is possible to edit the instance labels directly in the UI without needing to go through that process. This update also includes an improved UI for adding remote Databases to PMM.
+[IMAGE HERE]
+
+#### Connecting Services and Nodes  
+PMM-11963: It used to be difficult to figure out which Service was operating on which Node in PMM, but now things are easier. This new release lets you view a list of Services running on Nodes, along with the Node's name where the Service is being used. You can even filter by these names to make it simpler to navigate.
+[IMAGE HERE]
+
+#### Cluster view for Inventory [Technical Preview]
+PMM-12378: Understanding the structure of  your Inventory is important. In this release, we starting the experimental feature with the ability to see all your Services grouped by Cluster so users can understand inventory more easily. 
+[IMAGE HERE]
+
+ 
+
+### MongoDB backup monitoring 
+PMM-12384: Now, PMM not only allows you to perform MongoDB backups from the UI, but it also displays backup status metrics. Additionally, we are shipping the first version of the Alert Template to notify you of any failed backups.
```

Comment:
It would be useful to add a link to the documentation of the new backup status metrics. I guess this doc will be included as part of https://jira.percona.com/browse/PMM-12484. I think important to link here, so users can learn more. Thanks!